content/rosa/external-auth/index.md

```diff
@@ -0,0 +1,545 @@
+| --issuer-audiences | Comma separated list of OIDC token audience values | 
+| --claim-mapping-username-claim | Claim within the OIDC token containing the username | 
+| --claim-mapping-groups-claim | Claim within the OIDC token containing group membership | 
+| --console-client-id | OAuth Client ID to associate with the OpenShift Web Console | 
+| --console-client-secret | OAuth Client Secret to associate with the OpenShift Web Console. This option is not required, however is recommended |  
+
+Retrieve the OpenShift Authentication call back URL.  Save the output of this value, you will need it later.
+
+```bash
+echo $(rosa describe cluster --cluster $ROSA_CLUSTER_NAME -o json | jq -r '.console.url')/auth/callback
+```
+
+Example output:
+
+```
+https://console-openshift-console.apps.rosa.kmc-external.ya62.p3.openshiftapps.com/auth/callback
+```
+
+### Microsoft Entra Example
+
+#### 1. Register a new application in Azure Entra ID
+You need to create the Azure AD application itself. To do so, login to the Azure portal, and navigate to [App registrations blade](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade), then click on "New registration" to create a new application.
+
+![Azure Portal - App registrations blade](./images/azure-portal_app-registrations-blade.png)
+
+Provide a name for the application, for example `<cluster-name>-auth`. Select "Web" from the Redirect URI dropdown and fill in the Redirect URI using the value of the OAuth callback URL you retrieved in the previous step. Once you fill in the necessary information, click "Register" to create the application.
+
+![Azure Portal - Register an application page](./images/azure-portal_register-an-application-page.png)
+
+Then, click on the "Certificates & secrets" sub-blade and select "New client secret". Fill in the details request and make note of the generated client secret value, as you'll use it in a later step. You won't be able to retrieve it again.
+
+![Azure Portal - Certificates & secrets page](./images/azure-portal_certificates-secrets-page.png)
+![Azure Portal - Add a Client Secret page](./images/azure-portal_add-a-client-secret-page.png)
+![Azure Portal - Copy Client Secret page](./images/azure-portal_copy-client-secret-page.png)
+
+Then, click on the "Overview" sub-blade and make note of the "Application (client) ID" and "Directory (tenant) ID". You'll need those values in a later step as well.
+![Azure Portal - AD Overview page](./images/azure-portal_register-an-application-overview.png)
+
+#### 2. Configure optional claims
+
+In order to provide OpenShift with enough information about the user to create their account, we will configure Azure AD to provide two optional claims, specifically "email" and "preferred_username" when a user logs in. For more information on optional claims in Azure AD, see [the Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims).
```

Comment:
We cannot use `preferred_username` and should remove that until the product is able to support this.